strings/tcg/set_two.json

```diff
@@ -267,12 +267,12 @@
 			"id": "shrike",
 			"name": "Xenomorph Shrike",
 			"desc": "It is unknown why Shrikes are able to lead a Hive, but their hives are always much smaller than a Queen's.",
-			"rules": "{$Hivemind}. If Xenomorph Queen would be destroyed, you may re-equip Xenomorph Queen on this card instead, once per game.",
+			"rules": "{$Hivemind}. Tap: Add Xenomorph Queen from your discard pile to your hand.",
```

Comment:
shouldnt it be on this guy specifically? seems weird flavorwise

Reply:
I don't disagree on the flavor front, but I think adding it onto the hand rather than putting it on itself is more fun. Gives for more options for the player, lets you protect the Shrike with the unit that you equipped the Queen to, etc.